drake/systems/plants/test/testIKMoreConstraints.cpp

```diff
@@ -0,0 +1,221 @@
+#include <drake/RigidBodyIK.h>
+#include <drake/RigidBodyManipulator.h>
+#include <drake/RigidBodyConstraint.h>
+
+#include <drake/IKoptions.h>
```

Comment:
These are going to refer to the headers in build/include. This is not what we want. They should be relative to the current directory, or in folders in drake-distro/drake included using CMake's include_directories.